src/components/Header/Header.jsx

```diff
@@ -34,6 +34,9 @@ export default () => {
             <NavItem>
               <NavLink href="/media">Media</NavLink>
             </NavItem>
+            <NavItem>
+              <NavLink href="/AdventureSite">Explore</NavLink>
```

Comment:
The following link returns `404` and I guess this link should not be included till the moment the Adventure site is live.

Reply:
The AdventureSite is currently hosted at https://terasology.org/AdventureSite/ so I'd recommend using that link for the time being.